src/transactions.rs

```diff
@@ -532,8 +453,9 @@ impl<Pk: MiniscriptKey + ToPublicKey> Satisfier<Pk> for RevaultInputSatisfier<Pk
         None
     }
 
-    fn check_after(&self, csv: u32) -> bool {
-        self.sequence == csv
+    fn check_older(&self, csv: u32) -> bool {
+        assert!((csv & (1 << 22) == 0));
```

Comment:
Do we really want to panic here if the check fails?

Reply:
Absolutely, if this bit is set then our whole logic is fucked (see the body of the function, which doesn't even handle this case).

We don't produce such Script anyway https://github.com/re-vault/revault/pull/15/files#diff-5981a850fcf9460704456c506b006dc8R134-R139 and if we come across one it'd do more harm to operate with the wrong logic and assumptions than just crashing straight.

Reply:
An argument in favor of asserting is also an explicitness toward the reviewer: this should not happen, please a) review under this assumption b) make sure this assumption holds.

But for Rust it's an argument for `debug_assert!()` i guess.